### ISSUE TITLE
Feature request: CSV (or DSV) rendering

### ISSUE THREAD

**Issue by e13mort** [JakeWharton/picnic#79]
Thank's for the great library. I use it in some of my projects and very happy with results (especially combined with Mosaic).

In one of my projects I've faced with requirement to output data as CSV along with human-readable tables. I've came up with my 
 own solution (not a rocket science) and got thinking that there could be some CSV related API in Picnic.

- CSV (or more general DSV) format has no different in meaning from ASCII tables
- Such new rendering API could benefit from existing Mosaic API (rows, columns, cell, columnSpan)
- On the other side new API could contradict the single responsibility principle of the library (only ASCII tables)

So my questions are:
- what do you think about supporting CSV?
- if it works will you add that API in some future?
- may I help with this?

Thank you

**Comment by JakeWharton** [comment]
I don't think it makes sense to include directly, but rather as a companion library. I personally don't use CSV or DSV and don't feel qualified to maintain such a thing. I made the mistake [in my last table library](https://github.com/JakeWharton/flip-tables/) of adding support for `ResultSet` conversion for databases, and regretted it more than once.
